torchtitan/experiments/simple_fsdp/llama3/parallelize.py

```diff
@@ -30,6 +30,8 @@
     # used to compute the scaling factor for quantization.
     torch.ops.aten.max.default,
     torch._higher_order_ops.flex_attention,
+    torch.ops.torch_attn._varlen_attn,
+    torch.ops.torch_attn._varlen_attn_backward,
```

Comment:
You don't need to add the backward. SAC only save the ops that are called in the forward.